docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,374 @@
+number of changing client devices, is very much welcome. Depending on the feedback,
+we might enable this feature by default in a future release.
+
+Public key in respondd data (optional)
+======================================
+
+If desired, the fastd public key of a node can be included in the respondd nodeinfo data,
+faciliating the correlations of VPN peers and nodes. As the VPN key is transmitted unencrypted
+in the fastd handshake, this would theoretically allow an ISP to determine which nodes
+are operated behind which internet line. Therefore, this feature must be enabled explicitly
+by setting *mesh_vpn.pubkey_privacy* to ``false`` in *site.conf*.
+
+B.A.T.M.A.N. V (experimental)
+=============================
+
+When using batman-adv compat 15, it is now possible to switch to the new routing
+algorithm B.A.T.M.A.N. V (while the old algorithm is called B.A.T.M.A.N. IV) by
+setting *mesh.batman_adv.routing_algo* to ``"BATMAN_V"``. Note that the new routing
+algorithm is not backwards-compatible, so nodes using different algorithms can
+not interoperate.
+
+.. _releases-v2018.1-site-changes:
+
+Site changes
+************
+
+site.mk
+=======
+
+* Due to improved package dependency handling, the packages
+  *gluon-config-mode-core* and *gluon-setup-mode* do not need
+  to be listed explicitly in *site.mk* anymore; they will be
+  pulled in implicitly.
+* As mentioned above, we recommend to include the *ebtables-limit-arp*
+  feature flag.
+* We recommend to use *GLUON_FEATURES* for all Gluon packages, and rely on
+  *GLUON_SITE_PACKAGES* for non-Gluon (OpenWrt) package only, as explained
+  in :ref:`user-site-feature-flags`.
+
+site.conf
+=========
+
+When updating a site configuration from Gluon 2017.1.x, the following changes
+must be made:
+
+* .. code-block:: lua
+
+    domain_seed = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+
+  These 32 bytes of random data (encoded in hexadecimal) are used to seed a number
+  of site/domain specific random values that must be the same on all nodes of the
+  same mesh, but different for different meshes. The following command
+  can be used to generate such a random value:
+
+  .. code-block:: shell
+
+    echo $(hexdump -v -n 32 -e '1/1 "%02x"' </dev/urandom)
+
+  In multidomain setups, repeat this command for each domain.
+
+  At the moment, only the VXLAN ID for wired meshing is derived from the domain seed.
+
+* .. code-block:: lua
+
+    mesh = {
+      vxlan = true, -- or false
+      -- ...
+    },
+
+  In single domain setups, the *mesh.vxlan* option is mandatory now. It should be set to *true* in new
+  meshes. Existing setups should set it to *false* to retain compatibility with older versions of Gluon.
+
+  In multidomain setups, *mesh.vxlan* defaults to *true* and does not need to be set explicitly.
+  It can still be set to *false* for individual domains that should allow wired meshing with existing
+  setups.
```

Comment:
It can still be set to *false* for individual domains to allow backwards-compatible wired meshing within existing setups, which is also useful for migrating an existing mesh to a multidomain-capable firmware.